tools/lint/config/eslint.config.js

```diff
@@ -151,7 +151,7 @@ module.exports = {
       },
     },
     {
-      files: ['packages/**/*.spec.{ts,tsx}'],
+      files: ['chat/**/*.spec.{ts,tsx}', 'packages/**/*.spec.{ts,tsx}'],
```

Comment:
I think this should also include tools (while we're touching this line)
https://jira.mongodb.org/browse/LG-4093

Reply:
also: `'{chat,packages,tools}/**/*.spec.{ts,tsx}'`

Reply:
👍